### ISSUE TITLE
Browse Page: organizations

### ISSUE THREAD

**Issue by danmihaila** [OCHA-DAP/hdx-ckan#2001]
Based on https://docs.google.com/drawings/d/1hC8IA45gg-nAA5SeAx3YBrmJXrdeV02vKRoWstZ7RMs/edit
Implement the list of organizations
blocked by #1997 and #1998

**Comment by danmihaila** [comment]
Pinging @aalecs 
I think we can include the same snippet from ebola or search page for the moment...

**Comment by danmihaila** [comment]
@mbellotti - I think we should use the style used for datasets on ebola or colombia page. It should be quite easy to be used.

**Comment by aalecs** [comment]
The template was commited 2 weeks ago on 1998, but I didn't close it since there still are some things to implement, like the fancy in page navigation which wasn't a priority. 

For classes like mL30, by quickly checking what they do you can figure out what they mean. "mL" classes only apply margin-left for an example. For the important classes I either commented in the code, or sent a mail when they were introduced. They are referenced almost everywhere so they are defined in the header.css file.

**Comment by danmihaila** [comment]
I will list here the issues I currently see on dev and move this issue to sprint 46. @mbellotti  please provide estimates in our doc, so @cjhendrix  could adjust:
1. blue bar missing
2. action dropdown
3. show more/less not working
4. pagination is using a different anchor for section (probably Arty change the sections anchor names) and when u click on a page it will not show the org section

**Comment by danmihaila** [comment]
closing this one as @cjhendrix opened another issue #2073
